src/stdlib/Localization.ts

```diff
@@ -0,0 +1,89 @@
+import * as fs from "fs";
+import * as path from "path";
+import { promisify } from "util";
+const readFile = promisify(fs.readFile);
+
+import { Interpreter } from "../interpreter";
+import { BrsString, Callable, ValueKind, StdlibArgument, RoDeviceInfo } from "../brsTypes";
+import { XmlDocument } from "xmldoc";
+
+/**
+ * Supported locales in RBI.
+ * @see https://developer.roku.com/docs/references/brightscript/interfaces/ifdeviceinfo.md#getcurrentlocale-as-string
+ */
+let locales = new Set<string>([
```

Comment:
what about spanish?

Reply:
I just copied and pasted this list from the [Brightscript documentation](https://developer.roku.com/docs/references/brightscript/interfaces/ifdeviceinfo.md#getcurrentlocale-as-string).

Is it missing one? It has `"es_ES", // International Spanish`, is that what you were looking for?

Reply:
I must be blind, I didn't see it the first time :homerDisappears: